README.md

```diff
@@ -27,21 +27,37 @@ Follow the [pnpm Twitter account](https://twitter.com/pnpmjs) for updates.
 
 ## Background
 
-`pnpm` maintains a flat storage of all your dependencies in `node_modules/.store`. They are then symlinked wherever they're needed.
+`pnpm` maintains a flat storage of all your dependencies in `~/.store`. They are then symlinked wherever they're needed.
```

Comment:
Because hopefully we will use the same store structure as ied. Though maybe `.pnpm-store` is a good idea till the specs are not finished